### ISSUE TITLE
why set the code:  loss_cls = 0

### ISSUE THREAD

**Issue by hhaAndroid** [Microsoft/singleshotpose#8]
Hello, why is the class loss function set to 0? but the cross entropy loss function is annotated

#####loss_cls   = self.class_scale * nn.CrossEntropyLoss(size_average=False)(cls, tcls)
  loss_cls = 0

**Comment by hhaAndroid** [comment]
I probably know it.

**Comment by btekin** [comment]
Hello, for single object pose estimation, we don't solve the classification problem as we train a regressor on a single object category. However, for multi object pose estimation we also minimize the classification loss.

**Comment by hhaAndroid** [comment]
ok，I konw it,thank you
